x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/view/dev_console.tsx

```diff
@@ -15,82 +23,150 @@ import {
   EuiSpacer,
   EuiText,
 } from '@elastic/eui';
+import { useIsMounted } from '../../../components/hooks/use_is_mounted';
 import { useIsExperimentalFeatureEnabled } from '../../../../common/hooks/use_experimental_features';
 import { useUrlParams } from '../../../components/hooks/use_url_params';
 import {
-  Command,
   CommandDefinition,
-  CommandServiceInterface,
   Console,
   RegisteredConsoleClient,
   useConsoleManager,
 } from '../../../components/console';
 
 const delay = async (ms: number = 4000) => new Promise((r) => setTimeout(r, ms));
 
-class DevCommandService implements CommandServiceInterface {
-  getCommandList(): CommandDefinition[] {
-    return [
-      {
-        name: 'cmd1',
-        about: 'Runs cmd1',
-      },
-      {
-        name: 'get-file',
-        about: 'retrieve a file from the endpoint',
-        args: {
-          file: {
-            required: true,
-            allowMultiples: false,
-            about: 'the file path for the file to be retrieved',
-          },
-        },
+const getCommandList = (): CommandDefinition[] => {
```

Comment:
I think we already talk about this but, is this behind a feature flag?

Reply:
yes, it is.